packages/portfolio-contract/src/resolver/types.ts

```diff
@@ -82,6 +83,25 @@ export const PublishedTxShape: TypedPattern<PublishedTx> = M.or(
     {},
     {},
   ),
+  M.splitRecord(
+    {
+      type: M.or(TxType.IBC_FROM_AGORIC, TxType.IBC_FROM_REMOTE),
+      status: TxStatus.PENDING,
+    },
+    {
+      op: M.string(),
+      src: M.arrayOf(M.or(M.string(), [M.string(), M.any()])),
+      dst: M.arrayOf(M.or(M.string(), [M.string(), M.any()])),
```

Comment:
"destination" is abbreviated `dest` elsewhere. hm.

Reply:
I can spell it here as `dest` to do as the Romans do.  I plan on leaving `@agoric/orchestration` the way it is, though.